### PR TITLE
table-based q learner

### PR DESCRIPTION
**Pull request by simonyilunw**

### CODE REVIEW

File: rl/q_learner.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    return self.action
+
+
+  def nextAction(self, state, reward):
+    if self.exploration > random.random():
+      action = random.randint(0, self.num_actions-1)
+    else:
+      action = self.qtable[state].argsort()[-1]
+
+    self.train_iteration += 1
```
Comment (yukezhu) [inline rl/q_learner.py:51]: Can you move the training code from nextAction() to a new method, like updateModel()? During testing, you want to call nextAction() to only get actions without updating the Q values.

**Comment by simonyilunw** [comment]
Done!

**Comment by yukezhu** [comment]
So eGreedyAction() is not called in the script?

**Comment by yukezhu** [comment]
Merged. Thanks!

**Pull request merged**